app/assess/routes.py

```diff
@@ -138,3 +140,47 @@ def fund_round(fund_id: str, round_id: str):
         abort(404)
 
     return render_template("round.html", fund=fund, round=fund_round)
+
+
+@assess_bp.route("/fragments/upload_documents/")
+def upload_documents():
+    """
+    Render html page with json contains title & answer/url.
+    """
+    
+    uploaded_file_json = {
+        "name": "Digital Form Builder - Runner test-form-",
+        "questions": [
+            {
+                "question": "Upload documents page",
+                "fields": [
+                    {
+                        "key": "ocdeay",
+                        "title": "Python language page",
+                        "type": "file",
+                        "answer": "https://en.wikipedia.org/wiki/Python_(programming_language)",
+                    },
+                    {
+                        "key": "ocdeayx",
+                        "title": "JavaScript language page",
+                        "type": "file",
+                        "answer": "https://en.wikipedia.org/wiki/JavaScript",
+                    },
+                    {
+                        "key": "ocdeayy",
+                        "title": "C ++ language page",
+                        "type": "file",
+                        "answer": "https://en.wikipedia.org/wiki/C%2B%2B",
+                    },
+                ],
+            }
+        ],
+    }
+
+    json = uploaded_file_json["questions"][0]["fields"][0]
+
+    return render_template(
+        "macros/example_upload_documents.html",
+        title=json["title"],
+        url=json["answer"],
```

Comment:
💯 🔧 Great stuff Ram and a rapid fix, although we may want the page to take a list so we can render multiple files at once :)